evap/evaluation/templates/email_base.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html style="background-color: #495057;">
+
+<head>
+  <meta charset="utf-8"/>
+  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
+  <meta name="viewport" content="width=device-width; initial-scale=1; shrink-to-fit=no"/>
+  <title>EvaP</title>
+  <style>
+    @media (min-width: 768px) {
+      .spacer {
+        width: 15% !important;
+      }
+
+      .email-content {
+        width: 70% !important;
```

Comment:
just a question because i'm not sure: do you think it would be a good idea to set a max width here? for the website, the main content part has a max width so it doesn't span the whole screen in large browser windows.

Reply:
Yes thats much better. I used `max-width: 1000px` because `720px` seemed a bit less and `1000px` was the next threshold used on the website as well. Do you think that is ok?

Reply:
yes, seems reasonable